### ISSUE TITLE
[Screen Reader-Documentation] Name Property is not defined for the 'Print ' button showing on Documentation page

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2796]
**User Experience:**
Users with Visual disabilities might not able to understand the controls if Name Property is not defined. Like in the scenario Name Property is not defined for the Print button.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to url:-https://makecode.microbit.org/reference/basic/show-number
2-Navigate to the Print button showing on Documentation page
3-Verify that Name Property is defined for the Print button or not

**Actual Result**
Name Property is not defined for the 'Print ' button showing on Documentation page

**Expected Result**
Name Property should be defined for the 'Print ' button showing on Documentation page

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Note:**
Same issue appears with Microsoft link

**Suggested fix:**
Consider using the aria-labelledby property to bind the list to the element that contains the visual label. Expected results are:
The UIA Name property matches the visual label.
The relationship between the label and the list is programmatically exposed.
This approach avoids localization of identical strings. (The label does not have to be repeated on the list.)Code maintenance of accessibility properties is avoided in the case where the visual text label must be changed.
Note: The snippet below is a sample showing an unordered list of links. The example binds the list to the visual text label using the aria-labelledby property. 

**Refer Attachment:**
[MAS4.1.2_NotReadingName.zip](https://github.com/Microsoft/pxt/files/1248493/MAS4.1.2_NotReadingName.zip)

[MAS 4.1.2_NotreadingName.zip](https://github.com/Microsoft/pxt/files/1248509/MAS.4.1.2_NotreadingName.zip)

**Comment by veler** [comment]
Issue duplicated with [https://github.com/Microsoft/pxt/issues/2623](https://github.com/Microsoft/pxt/issues/2623)

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/reference/basic/show-number and issue is fixed now. Hence closing the bug
